pkg/imageverifier/bindir/bindir_test.go

```diff
@@ -176,8 +176,10 @@ func TestBinDirVerifyImage(t *testing.T) {
 
 		j, err := v.VerifyImage(ctx, "registry.example.com/image:abc", ocispec.Descriptor{})
 		assert.NoError(t, err)
```

Comment:
`if` works, but I'd recommend `require.NoError(t, err)` instead for consistency. `require.*` assertions break the execution of the test.

Reply:
hi @kzys! Thanks for the review :)

I assumed the next assertion was checking some values even if err was set. I checked the underlying function and this is not really the case. Hence, I followed your suggestion and edited the commit message.